### PR TITLE
Connection strings for other frameworks (UPM-5977)

### PR DESCRIPTION
**Pull request by moiznalwalla**
https://enterprisedb.atlassian.net/browse/UPM-5977

### CODE REVIEW

File: product_docs/docs/biganimal/release/using_cluster/02_connecting_your_cluster/index.mdx [hunk 1]
```diff
@@ -36,13 +36,37 @@ You can connect to your cluster using [`psql`](http://postgresguide.com/utilitie
 
 ## Recommended settings for SSL mode
 
-Different clients can have different default TLS/SSL modes (sslmode). For example, `psql` defaults to `prefer`, which means the client will attempt to establish a TLS connection but fall back to non-TLS if the server does not support it. In the `psql` example provided by EDB in the **Quick Connect** field, `sslmode` is explicitly set to `require`, which means the client will attempt a TLS connection and fail if the connection to the server can't be encrypted.
```
Comment (drothery-edb) [inline product_docs/docs/biganimal/release/using_cluster/02_connecting_your_cluster/index.mdx:39]: I don't think it is a bad thing to point out that sslmode is the parameter that governs the TLS/SSL modes. This was a subtle way of doing that. I would lean to keeping it in unless you think it is causing confusion.

File: product_docs/docs/biganimal/release/using_cluster/02_connecting_your_cluster/index.mdx [hunk 2]
```diff
@@ -36,13 +36,37 @@ You can connect to your cluster using [`psql`](http://postgresguide.com/utilitie
...
+For public connections and in most environments, EDB recommends setting `sslmode` to `verify-full`. This setting ensures that you connect to the server you specified and that the connection is encrypted.
 
-```
-psql -W "postgres://edb_admin@xxxxxxxxx.xxxxx.biganimal.io:5432/edb_admin?sslmode=verify-full&sslrootcert=/usr/share/ca-certificates/isrgrootx1.pem"
-```
+BigAnimal generates certificates with LetsEncrypt, a widely trusted certificate authority. Your client machine might already have a bundled CA certificate for LetsEncrypt, for example, at `/etc/ssl/certs/ca-certificates.crt` or `/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem`. If it doesn't, your client machine needs a [CA certificate for Let's Encrypt](https://letsencrypt.org/certificates/). Once the CA certificate is in place on your client machine, configure the `sslrootcert` parameter to its location and set the `sslmode` parameter to `verify-full` to verify the certificate to fully validate the connection.
+
+Connection string examples for common frameworks:
```
Comment (drothery-edb) [inline product_docs/docs/biganimal/release/using_cluster/02_connecting_your_cluster/index.mdx:45]: I changed this wording a bit, but I'm not sure it is correct. Should "common frameworks" be "common client drivers?"

File: product_docs/docs/biganimal/release/using_cluster/02_connecting_your_cluster/index.mdx (hunk 1 above)
Comment (moiznalwalla) [reply product_docs/docs/biganimal/release/using_cluster/02_connecting_your_cluster/index.mdx:39]: Added it back.

File: product_docs/docs/biganimal/release/using_cluster/02_connecting_your_cluster/index.mdx [hunk 3]
```diff
@@ -36,13 +36,37 @@ You can connect to your cluster using [`psql`](http://postgresguide.com/utilitie
 
 ## Recommended settings for SSL mode
```
[the remark this answers is not in the document]
Comment (moiznalwalla) [reply product_docs/docs/biganimal/release/using_cluster/02_connecting_your_cluster/index.mdx:37]: Added.

File: product_docs/docs/biganimal/release/using_cluster/02_connecting_your_cluster/index.mdx (hunk 2 above)
Comment (moiznalwalla) [reply product_docs/docs/biganimal/release/using_cluster/02_connecting_your_cluster/index.mdx:45]: "common database drivers"

File: product_docs/docs/biganimal/release/using_cluster/02_connecting_your_cluster/index.mdx [hunk 4]
```diff
@@ -32,17 +32,41 @@ You can connect to your cluster using [`psql`](http://postgresguide.com/utilitie
...
+- JDBC for Java
+
+   ```
+   jdbc:postgresql://xxxxxxxxx.xxxxx.biganimal.io:5432/edb_admin?&sslrootcert=/usr/share/ca-certificates/ca-cert_name.pem&sslmode=verify-full?user=edb_admin&password=$PWD
+   ```
+
+- Npgsql for DotNet Core
+   
+   ```
+   {{Host=xxxxxxxxx.xxxxx.biganimal.io;Port=5432;Username=edb_admin;Password=$PWD;Database=edb_admin;SslRootCert=/usr/share/ca-certificates/ca-cert_name.pem;SslMode=verify-full;}}
```
Comment (leonardoce) [inline product_docs/docs/biganimal/release/using_cluster/02_connecting_your_cluster/index.mdx:62]:
```suggestion
Host=xxxxxxxxx.xxxxx.biganimal.io;Port=5432;Username=edb_admin;Password=$PWD;Database=edb_admin;SslRootCert=/usr/share/ca-certificates/ca-cert_name.pem;SslMode=verify-full
```

File: product_docs/docs/biganimal/release/using_cluster/02_connecting_your_cluster/index.mdx [hunk 5]
```diff
@@ -32,17 +32,41 @@ You can connect to your cluster using [`psql`](http://postgresguide.com/utilitie
...
+   ```
+
+- ODBC for Windows
+
+   ```
+   Driver={PostgreSQL};Server=xxxxxxxxx.xxxxx.biganimal.io;Port=5432;Database=myDataBase;Uid=edb_admin;Pwd=$PWD;sslrootcert=C:\\ssl\\ca-certificate.pem;sslmode=verify-full;
```
Comment (leonardoce) [inline product_docs/docs/biganimal/release/using_cluster/02_connecting_your_cluster/index.mdx:68]:
```suggestion
Driver={PostgreSQL};Server=xxxxxxxxx.xxxxx.biganimal.io;Port=5432;Database=myDataBase;Uid=edb_admin;Pwd=$PWD;sslrootcert=C:\\ssl\\ca-certificate.pem;sslmode=verify-full;
```

**Review by leonardoce — CHANGES_REQUESTED** [review]

**Comment by leonardoce** [comment]
I agree with @moiznalwalla we need to test these connection strings to make sure they are working correctly.

**Review by leonardoce — DISMISSED** [review]

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**